### PR TITLE
Change api request to use json instead of html

### LINKED ISSUE #27: Api request should be done as json datatype instead of html
Currently all api requests are done as `text/html`. https://github.com/massiveart/web-js/blob/master/src/services/api.js#L19. Symfony will detect that and return errors as html instead of json. 

This would be a BC Break if someone uses `api.js` to return html so maybe we can change this only in 2.0.

### PR DESCRIPTION
**Pull request by alexander-schranz**
fixes #27

### CODE REVIEW

File: UPGRADE.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# 2.0.0
+
+## Api service dataType changed
+
+The api service will now use json dataType instead of html.
+So if expect html as answer you should use `$.ajax` instead.
```
Comment (thomasduenser) [inline UPGRADE.md]: I would change this to: So if HTML should be returned, you should use `$.ajax` instead.

**Comment by thomasduenser** [comment]
@alexander-schranz else ACK

**Comment by dominikmatt** [comment]
@alexander-schranz ACK

File: UPGRADE.md (hunk 1 above)
Comment (alexander-schranz) [reply UPGRADE.md]: @thomasduenser thx is changed.

**Pull request merged**